### PR TITLE
Migrate CI checks from CircleCI to GitHub actions

### LINKED ISSUE #6133: Migrate CI checks from CircleCI to GitHub actions
Currently Robolectric is still using CircleCI for PR checks.
Should we migrate to GitHub actions ？

### PR DESCRIPTION
**Pull request by LitterSun**
Fixes #6133

### CODE REVIEW

File: .github/workflows/check_java_formatting.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11.0.8
+
+      - name: Show all branch
+        run: |
+          git branch
+          git branch -r
+
+      - name: Download google-java-format 1.8
```
Comment (utzcoz) [inline .github/workflows/check_java_formatting.yml:29]: The master has upgraded it to 1.9.

Comment (LitterSun) [reply .github/workflows/check_java_formatting.yml:29]: Get

**Comment by LitterSun** [comment]
@hoisie could you help to review this PR? Thanks.

**Comment by hoisie** [comment]
Thank you so much for doing this. What's the best way to test this before merging it? I just want to make sure that the existing CircleCI checks can continue to run until these Github checks stabilize.

**Comment by LitterSun** [comment]
> Thank you so much for doing this. What's the best way to test this before merging it? I just want to make sure that the existing CircleCI checks can continue to run until these Github checks stabilize.

I've tried testing it before, and you can see [https://github.com/LitterSun/robolectric/actions](https://github.com/LitterSun/robolectric/actions), I also tested some error cases.
I agree with keeping CircleCI checks until these Github checks stabilize.

File: .github/workflows/check_java_formatting.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
+name: Check java formatting
```
Comment (hoisie) [inline .github/workflows/check_java_formatting.yml:1]: nit: `Java`

File: .github/workflows/tests_for_sdks.yml [hunk 3]
```diff
@@ -0,0 +1,94 @@
+name: Tests for sdks
```
Comment (hoisie) [inline .github/workflows/tests_for_sdks.yml:1]: nit: just call this `Tests`?

File: .github/workflows/check_java_formatting.yml (hunk 2 above)
Comment (LitterSun) [reply .github/workflows/check_java_formatting.yml]: Got

File: .github/workflows/tests_for_sdks.yml (hunk 3 above)
Comment (LitterSun) [reply .github/workflows/tests_for_sdks.yml]: Of course

**Comment by hoisie** [comment]
Thanks again! Can't wait to get this up and running!

**Pull request merged**